core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -478,36 +482,45 @@ func makeEmptyTransaction(keyStore KeyStore, nonce uint64, gasLimit uint64, gasP
 	return keyStore.SignTx(fromAddress, tx, chainID)
 }
 
-func saveReplacementInProgressAttempt(db *gorm.DB, oldAttempt EthTxAttempt, replacementAttempt *EthTxAttempt) error {
+const insertIntoEthTxAttemptsQuery = `
+INSERT INTO eth_tx_attempts (eth_tx_id, gas_price, signed_raw_tx, hash, broadcast_before_block_num, state, created_at, chain_specific_gas_limit, tx_type, gas_tip_cap, gas_fee_cap)
+VALUES (:eth_tx_id, :gas_price, :signed_raw_tx, :hash, :broadcast_before_block_num, :state, NOW(), :chain_specific_gas_limit, :tx_type, :gas_tip_cap, :gas_fee_cap)
+RETURNING *;
+`
+
```

Comment:
You could probably move these inserts into `bulletprooftxmanager/orm.go`

Reply:
(Since they're used in tests too)

Reply:
Good suggestion